modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -3483,6 +3483,59 @@ definition {
 			actionLabel = "Update Action Label");
 	}
 
+	@description = "Verify creating a Commerce Product entry triggers an action to update the Commerce Product entry"
+	@priority = 5
+	test CanUpdateCommerceProductEntryAfterCreatingCommerceProductEntry {
+		property portal.acceptance = "true";
+
+		task ("Given Commerce Product object and given an action to update a Commerce Product entry after creating a Commerce Product entry") {
```

Comment:
`Given an action to update a Commerce Product entry after creating a Commerce Product entry`

We don't need to include Commerce Product since that's available by default in DXP.

Reply:
Looks like we'll need to resolve a merge conflict as well.